lib/system_win32.c

```diff
@@ -267,4 +267,17 @@ HMODULE Curl_load_library(LPCTSTR filename)
 #endif
 }
 
+bool Curl_win32_impersonating(void)
+{
+  bool ret = FALSE;
+#ifndef CURL_WINDOWS_APP
```

Comment:
not sure if this code should be disabled for win app. Perhaps, I should enable it?